src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java

```diff
@@ -24,6 +26,11 @@ public UserId createUser(User user) {
 		userId.setId(userIdDb);
 		return userId;
 	}
-
 	
+	@Override
+	public java.util.List<User> getUsers() {
+		java.util.List<User> response=repositoryApplication.findAll();

```

Comment:
Same scenario with this response list object, please add "Please add "java.util.List" like a import.